### ISSUE TITLE
Agents not getting email updates on tickets.

### ISSUE THREAD

**Issue by philbertphotos** [osTicket/osTicket#5047]
I have an issue that has cropped up, we just noticed after the upgrade.  When the client replies to the ticket, the assigned agent does not receive the update notification.
Strange enough we do get other notices like new tickets but its not consistent.

Server Information
osTicket Version | v1.12.2 (a5d898b)     —  Up to date
Web Server Software | Apache/2.4.18 (Ubuntu)
MySQL Version | 5.7.26
PHP Version | 7.2.2

Everything is turned on for alerts basically unchanged since before the upgrade 
What should I be looking for? 
The only modification was with the Client interface (modified)
/includes/client (all files just about)
offline.php
index.php
open.php

**Comment by JediKev** [comment]
@philbertphotos

Might be related to a bug fix we merged. Please read the Important Note on our blog:
https://osticket.com/blog/2019/07/23/osticket-v1-12-1-released/

If the issue still persists please let use know and we’ll look further into this.

Cheers.

**Comment by philbertphotos** [comment]
Ok does this also have to be set? It seems it no longer uses the default account with out being set. Annoying.
![image](https://user-images.githubusercontent.com/2892474/64955414-2d482280-d856-11e9-837a-11658bebf95c.png)

**Comment by JediKev** [comment]
@philbertphotos 

Nah, you shouldn't have to set that for it to work, if it's set to Department Email it should use the Department Email and if that's not set it should use the Default System Email. I would go through your email settings and ensure all are working properly.

Cheers.
